src/pybamm/models/submodels/interface/kinetics/diffusion_limited.py

```diff
@@ -12,7 +12,7 @@ class DiffusionLimited(BaseInterface):
 
     Parameters
     ----------
-    param :
+    self.param :
```

Comment:
The input argument does not change
```suggestion
    param :
```